specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/roleAssignments.json

```diff
@@ -0,0 +1,401 @@
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorContract"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "RoleAssignments"
+        ],
+        "operationId": "RoleAssignments_GetRoleAssignmentById",
+        "description": "Get role assignment by role assignment Id.",
+        "x-ms-examples": {
+          "Get role assignment information": {
+            "$ref": "./examples/RoleAssignments_GetRoleAssignmentById.json"
+          }
+        },
+        "consumes": [],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/RoleAssignmentIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success response.",
+            "schema": {
+              "$ref": "#/definitions/RoleAssignmentDetails"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorContract"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "RoleAssignments"
+        ],
+        "operationId": "RoleAssignments_DeleteRoleAssignmentById",
+        "description": "Delete role assignment by role assignment Id.",
+        "x-ms-examples": {
+          "Delete role assignment": {
+            "$ref": "./examples/RoleAssignments_DeleteRoleAssignmentById.json"
+          }
+        },
+        "consumes": [],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/RoleAssignmentIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ScopeParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success response."
+          },
+          "204": {
+            "description": "No Content."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorContract"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ErrorContract": {
+      "title": "Error details.",
+      "description": "Contains details when the response code indicates an error.",
+      "type": "object",
+      "properties": {
+        "error": {
+          "description": "The error details.",
+          "$ref": "#/definitions/ErrorResponse"
+        }
+      }
+    },
+    "ErrorResponse": {
```

Comment:
This has conflict with "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse", please use the latter one .

Reply:
Thanks @jianyexi for relative path to exact type. I get the error:

```UnhandledPromiseRejectionWarning: Error: '$.definitions.ErrorResponse.title' has incompatible values```